Add support for Organization Unit field

tdabasinskas opened the pull request:
Adding support for a new metric `ssl_cert_subject_organization_units`, which lists the organizations units specified via `OU` field on the certificate.

While I'm on that, I did the following cleanup as well:
- Add `.gitignore` file.
- Fixed the example target, as it was not working without `https://` prefix.
- Remove redundant commas before and after label values (let me know if there was an actual reason these were added).

ribbybibby commented on ssl_exporter.go:
```diff
@@ -119,6 +125,7 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 		subject_emails := cert.EmailAddresses
 		subject_ips := cert.IPAddresses
 		serial_no := cert.SerialNumber.String()
+		ous := cert.Subject.OrganizationalUnit
```
Could you change this variable to `subject_ous`? Just for the sake of parity with other vars.

tdabasinskas commented:
Thanos for a quick response, @ribbybibby.

I've updated the variable name as requested. As for these commas, thanks for the link - I wasn't aware about this pattern before. I've dropped that commit so these extra commas are preserved.

ribbybibby commented:
Thanks @tdabasinskas.
